### PR TITLE
adding service name to cash receipt

### LINKED ISSUE #142: (feature request) Print the Service next to the Invoice Identifier on the Cash Payment Receipt
Our clients currently rely on the service of the bill to decide what services get rendered to a client.  It would be helpful to attache the service name next to (or underneath) the invoice identifier on the Cash Payment receipt so that the staff do not have to transcribe it each time.

### PR DESCRIPTION
**Pull request by DedrickEnc**
This PR adds a service name to a cash receipt and closes #142

Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through JSHint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

File: server/controllers/finance/reports/cash/index.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ const ReportManager = require('../../../../lib/ReportManager');
 const pdf = require('../../../../lib/renderers/pdf');
 const db = require('../../../../lib/db');
 
+const PatientInvoices = require('../../patientInvoice');
```
Comment (sfount) [inline server/controllers/finance/reports/cash/index.js]: I don't think this import is ever used.

File: server/controllers/finance/reports/cash/receipt.handlebars [hunk 2]
```diff
@@ -52,35 +52,47 @@
...
-        </th>
+
         <th class="text-right">{{translate "FORM.LABELS.AMOUNT"}}</th>
       </tr>
     </thead>
     <tbody>
       {{#each payment.items as |item| }}
         <tr>
           <td>{{item.reference}}</td>
+          {{#unless  ../payment.is_caution}}
```
Comment (sfount) [inline server/controllers/finance/reports/cash/receipt.handlebars]: Nice use of the `unless` helper.

File: server/controllers/finance/reports/cash/receipt.pos.handlebars [hunk 3]
```diff
@@ -70,7 +70,7 @@
 <!-- /if -->
 
 <h2 style="text-align : right;">{{translate "FORM.LABELS.TOTAL_PAID"}}: {{currency payment.amount payment.currency_id}}</h2>
-<h2 style="text-align : right;">{{translate "FORM.LABELS.TOTAL_BALANCE_REMAINING"}} : {{currency debtorTotalBalance payment.currency_id}}</h2>
+<h2 style="text-align : right;">{{translate "FORM.LABELS.TOTAL_BALANCE_REMAINING"}} : {{currency debtorTotalBalance enterprise.currency_id}}</h2>
```
Comment (jniles) [inline server/controllers/finance/reports/cash/receipt.pos.handlebars]: Nice catch :+1:

Comment (sfount) [inline server/controllers/finance/reports/cash/receipt.pos.handlebars]: Good catch. This balance should not be shown in anything but the enterprise currency unless a disclaimer is shown to the client with the exchange rate of today.

**Pull request merged**